content/docs/dvclive/api-reference/live/log_metric.md

```diff
@@ -0,0 +1,79 @@
+# Live.log_metric()
+
+```py
+ def log_metric(name: str, val: float):
+```
+
+## Usage
+
+```py
+from dvclive import Live
+
+live = Live()
```

Comment:
Not blocking so we can get this merged, but let's follow up on all these method snippets and use the context manager everywhere (and in this case, add `metrics.json` to the output).